open-company/open-company-web
# Redlines round one

card: https://trello.com/c/N6wrbezw

To test:
- check page layout with Zeplin
- switch category
- expand topics
